app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::API
+
+  def current_user
+    session[:current_user]
+  end
+
+  def user_authorized?
+    current_user&.id != nil
```

Comment:
I'd probably think of it as more of `if (current_user != nil) { return current_user.id != nil }`
See [this](https://en.wikipedia.org/wiki/Safe_navigation_operator#Ruby) for more info